doc/source/releases/0.10.0.rst

```diff
@@ -32,10 +32,19 @@ New IO modules
 
   * CEDIO - an alternative to spike2io
   * AxonaIO
-  * OpenEphysIO
+  * OpenEphysIO - handle the binary format
   * PhyIO
   * SpikeGLXIO
   * NWBIO
+  * MaxwellIO
+
+RawIO modules
+-------------
+
+Internal refactoring of the neo.rawio module about the channel grouping.
+There is a signal_stream filed that handle channel groups for signal.
+This enhanced the way annotation and array_annotaion are rendered at neo.io level.
```

Comment:
I am not sure if internal changes are interesting here, I thought the release notes are mostly for users, not developers. @apdavison What's your opinion here?

Reply:
I think it is important to track because spikeinterface depend on neo.rawio for wrapping reader. It is good to make it clear. No ?

Reply:
Ok, with this in mind it makes sense to mention it. I made a suggestion on improving the wording a bit.